observability/logging/logging-6.1/log6x-loki-6.1.adoc

```diff
@@ -0,0 +1,44 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[leveloffset=+1]
+[id="log6x-loki-6.1"]
+= Storing logs with LokiStack
+:context: log6x-loki-6.1
+
+toc::[]
+
+You can configure a `LokiStack` CR to store application, audit, and infrastructure-related logs.
+
+include::snippets/log6x-loki-statement-snip.adoc[leveloffset=+1]
+
+[id="prerequisites_{context}"]
+== Prerequisites
+
+* You have installed the {loki-op} by using the CLI or web console.
+* You have a `serviceAccount` in the same namespace in which you create the `ClusterLogForwarder`.
+* The `serviceAccount` is assigned `collect-audit-logs`, `collect-application-logs`, and `collect-infrastructure-logs` cluster roles.
+
+[id="setup_{context}"]
+== Core Setup and Configuration
+*Role-based access controls, basic monitoring, and pod placement to deploy Loki.*
```

Comment:
I don't think this should be bolded and should be a complete sentence if introducing a section.